lib/engine/corporation.rb

```diff
@@ -247,7 +247,15 @@ def holding_ok?(share_holder, extra_percent = 0)
     end
 
     def all_abilities
-      @companies.flat_map(&:all_abilities) + @abilities
+      all_abilities = @companies.flat_map(&:all_abilities) + @abilities
+      if owner.respond_to?(:companies)
+        all_abilities += owner.companies&.flat_map do |c|
+          c.all_abilities.select do |a|
+            a.when.to_s.include? 'owning_player'
+          end
+        end
+      end
+      all_abilities
```

Comment:
@michaeljb this code includes all owner privates with `owning_player` abilities in `all_abilities` for a corporation. Do you agree with this approach (1880 private abilities work for all our corporations) or is there another way its supposed to work?